src/views/tenant/administration/Tenants.js

```diff
@@ -113,7 +113,7 @@ const columns = [
     name: 'Actions',
     center: true,
     cell: (row) => (
-      <Link to={`/tenant/administration/tenants/Edit?TenantFilter=${row.defaultDomainName}`}>
+      <Link to={`/tenant/administration/tenants/Edit?tenantDomain=${row.defaultDomainName}`}>
```

Comment:
Why are we changing to `tenantDomain` when we use TenantFilter basically everywhere else?

Reply:
Stick to tenantFilter plz,

Reply:
So the pattern is usually `TenantFilter` is the URL param and `tenantDomain` is the JS variable.